### DISCUSSION TITLE
addPeak2GeneLinks only calculates p2gs in cis?

### DISCUSSION THREAD

**Discussion by RegnerM2015** [GreenleafLab/ArchR#899] [Questions / Documentation]
Hi @rcorces and @jgranja24 

Does the peak-to-gene analysis only focus on peaks that are on the same chromosome as the genes they link to? In other words, is the analysis restricted to links in cis? 

I noticed that all of my peak-to-genes links were cis relationships and none were trans (different chromosomes).

**Comment by rcorces** [comment]
yes - this is implied by `maxDist`
https://www.archrproject.com/reference/addPeak2GeneLinks.html
